resources/platform/linux/README.md

```diff
@@ -0,0 +1,23 @@
+In order to update the libradamsa shared object, clone the radamsa repo
+available at: 
+
+https://gitlab.com/akihe/radamsa
+
+Add the following method to the file c/lib.c
```

Comment:
Ideally include the following as a patch, or at the very least something like echo changes >> c/lib.c

Reply:
Done